android/src/app/organicmaps/SplashActivity.java

```diff
@@ -93,12 +102,14 @@ protected void onResume()
     mBaseDelegate.onResume();
     if (mCanceled)
       return;
-    if (!Config.isLocationRequested() && !LocationUtils.isLocationGranted(this))
+    if (!Config.isLocationRequested() && ActivityCompat.checkSelfPermission(this, ACCESS_COARSE_LOCATION)
+        != PackageManager.PERMISSION_GRANTED)
     {
-      ActivityCompat.requestPermissions(this, new String[]{
+      Logger.d(LOCATION_TAG, "Requesting location permissions");
```

Comment:
```suggestion
      Logger.i(LOCATION_TAG, "Requesting location permissions");
```
Wouldn't it be helpful in production?

Reply:
I updated to INFO.